recipes/opencv/run_test.sh

```diff
@@ -0,0 +1,5 @@
+g++ $RECIPE_DIR/test.cpp -I$PREFIX/include  -L$PREFIX/lib -o test
+[ $(./test) != "$PKG_VERSION" ] && exit 1
+
+[ $(python -c 'import cv2; print(cv2.__version__)') != "$PKG_VERSION" ] && exit 1
+
```

Comment:
Ahhhhhhhhh why is nothing ever simple. Above I used `python` - which is clearly a faux pas -  I should have used `$PYTHON`. No problem! I'll just fix that now. Oh wait, now I get an error that `/Users/pts08/miniconda/envs/**_build**/bin/python` does not exist... In a testing environment...